assets/stylesheets/common/campaign.scss

```diff
@@ -314,15 +314,15 @@ html:not(.mobile-view) .subscriptions-campaign-topic-footer .campaign-banner {
 
 $particles: 50;
 $width: 500;
-$halfwidth: $width / 2;
+$halfwidth: calc($width / 2);
```

Comment:
Does `calc()` (a css function) work with sass `+` and `-` operators used below?

Reply:
Yes, it still works, I see the animations (and the values in the generated CSS): 

<img width="757" alt="image" src="https://github.com/discourse/discourse-subscriptions/assets/368961/c4e6fb2e-2f01-4c94-ab5a-d7603b9fbf8b">

In the latest commit, I removed some redundant duplication for this effect. There was a second instance of the styles for when the element is in the sidebar, but it's not necessary. I also reduced the number of effects to 2, from 5.

Screenshot of this working in this sidebar implementation: 

<img width="418" alt="image" src="https://github.com/discourse/discourse-subscriptions/assets/368961/3d2aa5c4-83c0-40bb-b05d-ef41474b7a84">